modules/imgcodecs/src/grfmt_webp.cpp

```diff
@@ -331,6 +335,110 @@ bool WebPEncoder::write(const Mat& img, const std::vector<int>& params)
     return size > 0;
 }
 
+bool WebPEncoder::writemulti(const std::vector<Mat>& img_vec, const std::vector<int>& params)
+{
+    Animation animation;
+    animation.frames = img_vec;
+    int timestamp = 100;
```

Comment:
This is the reason why I think writemulti should not support AVIF and WebP. Those are not multi-page formats (and yes, I added it for AVIF but now think that it should be removed)